src/DB_Command.php

```diff
@@ -456,6 +456,177 @@ public function tables( $args, $assoc_args ) {
 		}
 	}
 
+	/**
+	 * Display the database name and size as ASCII table.
+	 *
+	 * Displays the Display the database name and size as ASCII table for `DB_NAME` specified in wp-config.php.
```

Comment:
"Displays the Display the database name" seems like a typo.

In this extended description, it'd be worthwhile to mention the default size format (which is configurable).